pydatastructs/linear_data_structures/tests/test_arrays.py

```diff
@@ -27,6 +28,7 @@ def test_DynamicOneDimensionalArray():
     A.append(2)
     A.append(3)
     A.append(4)
+    assert str(A) == '[1, 2, 3, 4]'
```

Comment:
```suggestion
```

Reply:
Test for https://github.com/codezonediitj/pydatastructs/pull/208/files#diff-df3f437c897b2969b957400929ff64f7R38

Reply:
I am sorry it is directing me to array.py and I cant understand what I am supposed to do, can you please elaborate? thanks

Reply:
What's the output when, `A._data`  attains `[4, None, None]`(in the tests below).

Reply:
Done. Thanks for pointing out.